packages/native/src/components/list/SingleDropdownList.js

```diff
@@ -192,7 +192,10 @@ class SingleDropdownList extends Component {
 				style={this.props.style}
 			>
 				{
-					[...selectAll, ...this.state.options].map(item => (
+					[
+						...selectAll,
+						...this.state.options.filter(item => String(item.key).length),
```

Comment:
Using `String(item.key).trim().length` should give better results

Reply:
yes, use `.trim()`